nrf70_bm_lib/docs/source/nrf70_bm_lib.rst

```diff
@@ -98,3 +98,46 @@ The library driver code execute in the following contexts:"
   .. note::
      In the reference implementation for Zephyr tasklet work is offloaded to Zephyr kernel workqueues.
 
+Optimzing scan operation
+########################
+
+The nRF70 Series BM library provides a single API to perform a Wi-Fi scan operation.
+The scan operation is optimized to provide a wide range of scan configuration parameters.
+
+Please see `Optimizing scan operation <https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/protocols/wifi/scan_mode/scan_operation.html>`_ for more information.
+
+nRF70 Series device states
+##########################
+
+The power save state of the nRF70 Series device is described through a combination of the physical power state of the logic or circuits and the logical functional state as observed by 802.11 protocol operations.
+
+Power state
+***********
+
+The nRF70 Series device can be in one of the following power states:
+
+* **Active:** The device is **ON** constantly so that it can receive and transmit the data.
+* **Sleep:** The device is **OFF** to the majority of the blocks that cannot receive and transmit the data.
+  In this state, the device consumes low power (~15 µA).
+  Real-time Clock (RTC) domain, RF retention memory, and firmware retention memory are powered **ON** to retain the state information.
+* **Shutdown:** The device is completely powered **OFF**.
+  In this state, the device consumes very low power (~2 µA) and does not retain any state information (apart from the values in the OTP memory).
+  The device will only respond to a BUCKEN assertion to wake from the Shutdown state.
+
+.. note::
+
+  The **Sleep** state is only applicable when ``CONFIG_NRF_WIFI_LOW_POWER`` is enabled.
+
+The nRF70 Series transition to and from the Shutdown state is automatically managed by the nRF Wi-Fi driver.
+When the network interface is brought down, the nRF Wi-Fi driver puts the nRF70 Series device in Shutdown state.
+When the network interface is brought up, the nRF Wi-Fi driver puts the nRF70 Series device in Active state.
```

Comment:
Same comment here. Instead of network interface, should we refer to BM initialization functions?